### ISSUE TITLE
When using chat-radius, /me is not included

### ISSUE THREAD

**Issue by justwheel** [EssentialsX/Essentials#118]
Honestly not sure if this is a bug or an enhancement, but if you're using the chat-radius feature of EssentialsChat, it will make chat limited to a certain radius, but players can still use `/me` to get around the local chat. It would be nice to have emotes included and confined to local chat just like normal messages.

**Comment by drtshock** [comment]
That should work :) Test and let me know how it goes.

**Comment by justwheel** [comment]
I just tried testing it and received some really weird behavior. So I tested with players and with OPs. When players in a radius of each other use `/me`, they cannot see their own messages, but players nearby can see their emotes. Operators can also see their messages.

As an operator, I tried using `/me` several times and I also could not see my emotes. Players outside the radius could not see them either, but players in my radius could see my emotes.

Hope this all makes sense!

![Screenshot from console](http://i.j-f.co/u/4fdc5a13240d6caa4a530aeb7d79a0a6.png)
